scripts/post-react.js

```diff
@@ -0,0 +1,10 @@
+const gulp = require("gulp");
+const replace = require("gulp-replace");
+
+gulp.start('examples_react', function() {
```

Comment:
Works just as well without the `gulp.start` wrapper. Simply use `gulp.src` directly.